src/js/player.js

```diff
@@ -768,8 +772,14 @@ class Player extends Component {
 
     this.fluid_ = !!bool;
 
+    if (isEvented(this)) {
```

Comment:
same here, isEvented check isn't needed.

Reply:
Fluid is called first time before the player is evented.